### ISSUE TITLE
Version 0.3.0 crashes with Optifabric+Optifine

### ISSUE THREAD

**Issue by floral-qua-floral** [Leo40Git/Duravision#2]
The game is unable to launch when Duravision 0.3.0 is installed alongside [Optifine](https://optifine.net/downloads) and [Optifabric](https://www.curseforge.com/minecraft/mc-mods/optifabric). The launcher creates a popup saying a crash has occurred, but no report is generated. It does create a log file, though.

Versions:
Fabric loader 0.8.3-build.196
Fabric API 0.10.8-build.310
Duravision 0.3.0
Optifabric 1.0.0 beta 8
Optifine HD U G1 preview 18

Log file:
[latest.log](https://github.com/Leo40Git/Duravision/files/4651904/latest.log)

The issue does not occur when using Duravision 0.2.0, so it looks like this has something to do with a recent change.

**Comment by Leo40Git** [comment]
Yeah, OptiFine kinda nukes everything from orbit and OptiFabric patches compatability in addition to loading the thing. I'll go yell at modmuss.

**Comment by Leo40Git** [comment]
See https://github.com/modmuss50/OptiFabric/issues/207.

**Comment by floral-qua-floral** [comment]
Ah, that makes sense. I've heard Optifine can be a bit wacky to try and maintain compatibility with, but in this case since the previous version didn't have any problems I figured this might hopefully not be _too_ much trouble. I've just gone and checked if disabling the potion overlay in Duravision's config might fix the issue, but it doesn't seem to.

Also, just as an extra thing I've noticed that I'm not sure is supposed to happen - it doesn't look like Duravision generates a config file on its own. One does generate if I use the Mod Menu mod and open the in-game config screen, but until I do that it doesn't look like a config file generates, even if I enter a world. Should I put up a separate issue report for this?

**Comment by Leo40Git** [comment]
Might've forgot to save the config after loading it, but no biggie since it's just the default options and the config screen always saves.
Also I'm changing how the config works in 0.4 so it's probably already fixed

**Comment by Leo40Git** [comment]
okay well modmuss isn't answering

In the meantime, if you use OptiFine for the performance enhancements, I recommend using [Lithium](https://www.curseforge.com/minecraft/mc-mods/lithium) and [Phosphor](https://www.curseforge.com/minecraft/mc-mods/phosphor) instead. Unlike OptiFine, they **_don't completely nuke the game and replace it._**
